gcloud-java-examples/pom.xml

```diff
@@ -22,6 +22,12 @@
       <artifactId>gcloud-java</artifactId>
       <version>${project.version}</version>
     </dependency>
+      <dependency>
```

Comment:
the jar target? Yes this is necessary in order to be able to run the gcloud-nio example. A possible alternative would be to create a completely different module (pom, class file, and documentation) just for the Stat example, and have that one write to a jar. That might be overkill.